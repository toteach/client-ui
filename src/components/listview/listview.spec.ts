import { shallowMount, Wrapper } from '@vue/test-utils';
import Vue from 'vue';

import TListView from './listview.vue';

// eslint-disable-next-line max-lines-per-function
describe('TListView', () => {
  let wrapper: Wrapper<Vue>;
  beforeEach(() => {
    wrapper = shallowMount(TListView);
  });

  it('is initialized', () => {
    expect(wrapper.name()).toBe('TListView');
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  it('renders correctly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
