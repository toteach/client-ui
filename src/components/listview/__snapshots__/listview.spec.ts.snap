// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TListView renders correctly 1`] = `
"<div class=\\"t-list-view\\">
  <!---->
  <header class=\\"t-list-view__header\\">
    <!---->
  </header>
  <section class=\\"t-list-view__items\\">
    <!---->
  </section>
  <section class=\\"t-list-view__pagination\\">
    <!---->
  </section>
  <footer class=\\"t-list-view__footer\\">
    <!---->
  </footer>
</div>"
`;
